errors/errorHandlers.js

```diff
@@ -5,9 +5,20 @@ exports.handleCustomErrors = (err, req, res, next) => {
 };
 
 exports.handlePsqlErrors = (err, req, res, next) => {
-  if (["22P02", "23502", "42601"].includes(err.code)) {
-    res.status(400).send({ msg: "Bad Request" });
-  } else next(err);
+  switch (err.code) {
+    case "22P02":
+      res.status(400).send({ msg: "Bad Request: Invalid Input" });
+      break;
+    case "23502":
+      res.status(400).send({ msg: "Bad Request: Missing Required Field" });
+      break;
+    case "42601":
+      res.status(400).send({ msg: "Bad Request: Syntax Error" });
+      break;
+    default:
+      next(err);
+      break;
```

Comment:
Great use of switch. Not sure about '42601' Syntax Error. It's usually happens because of some errors in the model not caused by users. Probably no need to handle it